packages/api-aco/src/utils/FolderLevelPermissions.ts

```diff
@@ -74,8 +74,9 @@ export class FolderLevelPermissions {
             // At the moment, we only want FLP to be used with identities of type "admin".
             // This temporarily addresses the issue of API keys not being able to access content, because
             // FLPs doesn't work with them. Once we start adding FLPs to API keys, we can remove this check.
+            // Also, if the CRUD method is called from a background task, the identity might not exist at all.
```

Comment:
Looks good, but let's do a quick refactor:

```
this.canUseFolderLevelPermissions = () => {
    const identity = this.getIdentity();

    // FLPs only work with authenticated identities (logged-in users).
    if (!identity) {
        return false;
    }
    // At the moment, we only want FLP to be used with identities of type "admin".
    // This temporarily addresses the issue of API keys not being able to access content, because
    // FLPs doesn't work with them. Once we start adding FLPs to API keys, we can remove this check.
    if (identity.type !== "admin") {
        return false;
    }

    return params.canUseFolderLevelPermissions();
};
```

Reply:
Thanks @adrians5j!